ovsdb: Fix query output when port and interface name differ

cathay4t opened the pull request:
For OVS like:

```
Bridge br0
  Port port2
      Interface eth2
          type: system
  Port port1
      Interface eth1
          type: system
```

The `nmstatectl show` is taking `port1` as OVS bridge port name.

The fix is to use interface name in database when only one interface
attatched(OVS bond should still use port name).

Integration test case included.

cathay4t commented:
Wait #2841 #2843

ffmancera commented on rust/src/lib/ovsdb/show.rs:
```diff
@@ -80,7 +80,19 @@ fn parse_ovs_bridge_conf(
         if let Some(ovsdb_port) = ovsdb_ports.get(port_uuid) {
             let mut port_conf = OvsBridgePortConfig::new();
-            port_conf.name.clone_from(&ovsdb_port.name);
+            if ovsdb_port.ports.len() == 1 {
```
Could you add a comment here explaining why this can be > 1? I know it is because of ovs-bond but it would be nice to remember it as ovs-bond is not so obvious..

cathay4t replied:
Comment included.

ffmancera reviewed (approved):
LGTM, just a nit about a comment

cathay4t committed:
ovsdb: Fix query output when port and interface name differ

For OVS like:

```
Bridge br0
    Port port2
        Interface eth2
            type: system
    Port port1
        Interface eth1
            type: system
```

The `nmstatectl show` is taking `port1` as OVS bridge port name.

The fix is to use interface name in database when only one interface
attatched(OVS bond should still use port name).

Integration test case included.

Signed-off-by: Gris Ge <fge@redhat.com>
